includes/polling/os/fortios.inc.php

```diff
@@ -1,3 +1,25 @@
 <?php
 
-$serial = snmp_get($device, 'fnSysSerial', '-OQv', 'FORTINET-FORTIANALYZER-MIB');
+use LibreNMS\RRD\RrdDefinition;
+
+$serial = snmp_get($device, 'FORTINET-CORE-MIB::fnSysSerial.0', '-Ovq');
```

Comment:
Please convert both of these to snmp_get_multi_oid() and also drop the MIB name from the oid and put it as an arg to the function.